recipe/meta.yaml

```diff
@@ -70,7 +70,7 @@ outputs:
         - nvcc --verbose test.cu
         - cmake ${CMAKE_ARGS} -S . -B ./build -G=Ninja
         - cmake --build ./build -v
-        - bash ./run_cmake_cuda_tests.sh  # [linux]
+        - bash ./run_cmake_cuda_tests.sh  # [linux and (x86_64 or aarch64)]
```

Comment:
Can we just skip the one test that fails instead of skipping all tests?

Reply:
We skipped all of them here: https://github.com/conda-forge/cuda-nvcc-impl-feedstock/blob/main/recipe/meta.yaml#LL158C43-L158C76

Reply:
Yeah, I thought all tests failed, but it seems like only one test failed for ppc64le.